### DISCUSSION TITLE
eqMac v1.1.0

### DISCUSSION THREAD

**Discussion by nodeful** [bitgapp/eqMac#527] [General]
# v1.1.0 - Major bug fixes + Privacy
* New much more stable driver - no more distortion
* Privacy Consent form - choose what data you want to share (Opt-out by default)
* Fixed crashes during Sleep/Wake-up cycles
* Fixed bug when Window would always open on every launch
* Fixed macOS 10.12 UI rendering issues
* Enabled Virtual device support (Capture adjusted audio with BlackHole or similar software)
* Added OTA (Over the Air) updates Opt-out
* Saving last Window position between Launches
* Added "Always on Top" mode for Window
* Added "Neither" option to Dock/StatusBar selection
* Added Native macOS look-a-like Status Bar icon
* Volume controls bug fixes
* Other misc bug fixes and improvements

<hr /><em>This discussion was created from the release <a href='https://github.com/bitgapp/eqMac/releases/tag/v1.1.0'>eqMac v1.1.0</a>.</em>

**Comment by sarendipitee** [comment]
Fantastic work @nodeful! Been using for a couple days and haven't noticed any of those distortion issues!

Now if you could tackle #486 you'd have a Pro subscriber for life!

**Comment by ssent1** [comment]
I second the stability of this version. 👍

After I ran it, the GUI was fuzzy. I added the lines:

```sh
<key>NSHighResolutionCapable</key>
  <true/>
```
to `/Applications/eqMac.app/Contents/Info.plist`

As expected, the app did not run. However, after removing the lines and rebooting the app, the GUI has been crystal clear on my Retina display.

I have no idea why or if it was my intervention that worked. Regardless, the app now looks great, especially in "Slider" mode.

The menubar icon looks like it might need a little more "pixel-perfect" loving in Illustrator. 
<img width="177" alt="eqmac-menubar-icon" src="https://user-images.githubusercontent.com/893313/126392146-be91d84b-d69c-4966-9983-03e4c9d9a797.png">

You have a ton on your plate. If you want help with this, please let me know.
